python/pyspark/sql/connect/function_builder.py

```diff
@@ -50,7 +53,9 @@ def _build(name: str, *args: ExpressionOrString) -> ScalarFunctionExpression:
 class FunctionBuilder:
     """This class is used to build arbitrary functions used in expressions"""
 
-    def __getattr__(self, name):
+    def __getattr__(
+        self, name: str
+    ) -> "Callable[[VarArg(ExpressionOrString)], ScalarFunctionExpression]":
```

Comment:
It would be better to avoid `VarArg`. It's deprecated anyway, and we already use `Protocols` for this purpose so let's be consistent.

Reply:
Ok, I will follow the examples in protocols.